src/citrine/resources/data_concepts.py

```diff
@@ -611,3 +613,19 @@ def filter_by_name(self, name: str, exact: bool = False,
             params=params,
         )
         return [self.build(content) for content in response["contents"]]
+
+    def delete(self, uid: Union[UUID, str], scope: str = 'id'):
+        """
+        Delete the element of the collection with ID equal to uid.
+
+        Parameters
+        ----------
+        uid: Union[UUID, str]
+            The ID.
+        scope: str
+            The scope of the uid, defaults to Citrine scope ('id')
+
+        """
+        path = self._get_path() + "/{}/{}".format(scope, uid)
+        self.session.delete_resource(path)
```

Comment:
Did we establish that this line throws an error if the attempted delete fails?

Reply:
yes, and it's tested here https://github.com/CitrineInformatics/nextgen-devkit/blob/master/integration-tests/tests/user_journeys_strike-team-data/test_delete.py